### PR TITLE
Change the fault members identifying way

### PR DESCRIPTION
**Pull request by sajithswa**
When same host having more than one instance and if one instance goes
down fault member map contain the host name. It is a bug. It must be
unique for each instance. Change identifier to mamber_id witch unique
one in conf file

### CODE REVIEW

File: components/org.apache.stratos.load.balancer/src/main/java/org/apache/stratos/load/balancer/endpoint/TenantAwareLoadBalanceEndpoint.java [hunk 1]
```diff
@@ -801,10 +804,11 @@ public void setSessionAffinity(boolean sessionAffinity) {
         private org.apache.axis2.clustering.Member currentMember;
         private Endpoint currentEp;
         private EndpointReference to;
-        private Map<String, Boolean> faultyMembers;
+        private Map<String, Boolean> faultyMembersIds;
```
Comment (imesh) [inline components/org.apache.stratos.load.balancer/src/main/java/org/apache/stratos/load/balancer/endpoint/TenantAwareLoadBalanceEndpoint.java]: Thanks @sajithswa for fixing this! Shall we rename map variable faultyMembersIds to faultyMemberIds?

File: components/org.apache.stratos.load.balancer/src/main/java/org/apache/stratos/load/balancer/endpoint/TenantAwareLoadBalanceEndpoint.java [hunk 2]
```diff
@@ -80,6 +80,9 @@
     
     private final static Pattern LAST_INT_PATTERN = Pattern.compile("[^0-9]+([0-9]+)$");
 
+    /*  Define property name to get member_id*/
+    private static final String memberIDKey = "member_id";
```
Comment (imesh) [inline components/org.apache.stratos.load.balancer/src/main/java/org/apache/stratos/load/balancer/endpoint/TenantAwareLoadBalanceEndpoint.java]: We already have a constant for "member_id": LoadBalancerConstants.MEMBER_ID. Would you mind removing this and use the existing?
Thanks

Comment (sajithswa) [inline components/org.apache.stratos.load.balancer/src/main/java/org/apache/stratos/load/balancer/endpoint/TenantAwareLoadBalanceEndpoint.java]: Never mind. I changed the code to use existing constants

File: components/org.apache.stratos.load.balancer/src/main/java/org/apache/stratos/load/balancer/endpoint/TenantAwareLoadBalanceEndpoint.java (hunk 1 above)
Comment (sajithswa) [inline components/org.apache.stratos.load.balancer/src/main/java/org/apache/stratos/load/balancer/endpoint/TenantAwareLoadBalanceEndpoint.java]: Sure.

**Comment by R-Rajkumar** [comment]
Seems something went wrong with this PR. Why there are 58 commits?

**Comment by sajithswa** [comment]
I got some unnecessary commits to my git repo. So I roll back those.

**Comment by R-Rajkumar** [comment]
I guess you can do the following;
1. take this diff
2. reset your local git repo branch (sajithswa:master) with apache master branch (apache:master)
3. apply the diff
4. send the PR

**Comment by R-Rajkumar** [comment]
Since you have sent a different PR with these changes, can you please close this PR?

**Comment by sajithswa** [comment]
Sure.

**Pull request closed without merging**